edgedb-tokio/src/tutorial.md

```diff
@@ -448,4 +448,46 @@ cloned_client
 
 Note that atomic transactions can often be achieved with links instead of transaction operations, which is both more idiomatic to EdgeDB and easier to use.
 
-For example, a wedding ceremony can in theory be seen as an atomic operation as it involves two instantaneous changes in state (from single to married) and should not have a state in between where Person A is married to Person B but Person B is still not yet married to Person A. However, this can be easily accomplished through links instead. One method to accomplish this might be to insert a `WeddingCertificate` with links to two `Person` objects each holding a government id number, with the `Person` type linking to a `WeddingCertificate` containing its government id number to pull up marriage status, the spouse's Person object, and so on.
\ No newline at end of file
+For example, a wedding ceremony can in theory be seen as an atomic operation as it involves two instantaneous changes in state (from single to married) and should not have a state in between where Person A is married to Person B but Person B is still not yet married to Person A. However, this can be easily accomplished through links instead. One method to accomplish this might be to insert a `WeddingCertificate` with links to two `Person` objects each holding a government id number, with the `Person` type linking to a `WeddingCertificate` containing its government id number to pull up marriage status, the spouse's Person object, and so on.
```

Comment:
I don't think we need this block in Rust tutorial. We can have it in some "transactions guide" in the normal EdgeDB docs and link there if you wish.